### PR TITLE
Moped Editor | Project File Uploads

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/5299
Closes https://github.com/cityofaustin/atd-data-tech/issues/5301

### To test, you will need to run the stack locally.
1. Run the Hasura cluster
2. Run the Moped API

### Running the Moped API
1. You will need to have your AWS variables exported, otherwise the API will fail to load.
2. Export one more environment variable: `export MOPED_API_UPLOADS_S3_BUCKET="atd-moped-editor-staging"`
3. Run: `FLASK_ENV=development flask run --without-threads`

### CODE REVIEW

File: moped-editor/src/utils/downloadFileAttachment.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            .catch(err => {
+              console.log("Error Downloading File: " + JSON.stringify(err));
+            });
+        } else {
+          // Error
+          console.log("Error: Unable to download file.");
+        }
+      })
+      .catch(err => {
+        debugger;
```
Comment (mddilley) [inline moped-editor/src/utils/downloadFileAttachment.js:41]: A debugger left in here

**Review by mddilley — APPROVED** [review]
Works just as expected! I was able to get everything running locally thanks to your notes in the PR and test upload and download. 🙏  

One thing that I noticed when choosing a file to upload is that the file uploads to the S3 bucket regardless of whether I click "Save" or not in the modal. I think that the user experience of being able to see the file progress is really cool, but I could see users selecting the wrong file here and there. Just thinking out loud, and maybe this is something that is already on your radar or not a huge concern as far as storage goes.

Either way, 🎉 !
Screen Shot 2021-03-12 at 11 31 47 AM

File: moped-editor/src/utils/downloadFileAttachment.js (hunk 1 above)
Comment (sergiogcx) [reply moped-editor/src/utils/downloadFileAttachment.js:41]: Aah, darn it. Let me get rid of it 🙏

Comment (sergiogcx) [reply moped-editor/src/utils/downloadFileAttachment.js]: Removed! 👍

**Comment by amenity** [comment]
Y'all I don't have time to run locally, but looks wonderful! I have a few design tweaks that I'll stick in a new issue.

**Review by amenity — APPROVED** [review]

File: moped-editor/src/utils/humanReadableFileSize.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * @param bytes Number of bytes.
+ * @param si True to use metric (SI) units, aka powers of 1000. False to use
+ *           binary (IEC), aka powers of 1024.
+ * @param dp Number of decimal places to display.
+ *
+ * @return Formatted string.
+ */
+export default (bytes, si=false, dp=1) => {
+    const thresh = si ? 1000 : 1024;
```
Comment (mateoclarke) [inline moped-editor/src/utils/humanReadableFileSize.js:12]: math

**Review by mateoclarke — COMMENTED** [review]
Really good stuff in here and glad to see it already working in staging!

**Pull request merged**